core/src/services/azblob/backend.rs

```diff
@@ -61,6 +61,9 @@ pub struct AzblobBuilder {
     endpoint: Option<String>,
     account_name: Option<String>,
     account_key: Option<String>,
+    server_side_encryption_customer_key: Option<String>,
```

Comment:
I recommend using `encryption_key`, `encryption_key_sha256`, and `encryption_algorithm` as configuration names to maintain consistency with the API.

![image](https://github.com/apache/incubator-opendal/assets/5351546/8f4cda62-44e8-4087-89db-d1592a17cfbe)

Reply:
Aligned names both in AzblobBuilder and AzblobCore